src/main/webapp/detailsScript.js

```diff
@@ -0,0 +1,97 @@
+const API_KEY = 'AIzaSyA4o8AqxFRAFJD_S5-cEXPlJMqVeGHhGLI';
+// To get results from UK publishing houses.
+const country = 'UK';
+
+// Obtain the book ID from the quert string. This is the Google Books API ID.
+let string = window.location.search;
+let id = string.substring(string.lastIndexOf('=') + 1);
+
+/**
+ * Function that is automatically triggered when the user is redirected to the bookDetails page.
+ * Extracts the Google Books API ID of the individual book from the URL and performs a GET request
+ * with that ID to obtain a JSON with all the details about that specific book.
+ */
+
+function getBookDetails() {
+  
+  let URL = `https://www.googleapis.com/books/v1/volumes/${id}`;
+  URL += `?key=${API_KEY}`;
+  URL += `&country=${country}`;
+  $.ajax({
+    url: URL,
+    type: 'GET',
+    success: function(result) {
+      console.log(result);
+      displayDetails(result);
+    },
+    error: function(error) {
+      alert(`Error ${error}. Please try again.`);
+    },
+  });
+}
+
+/**
+ * Completes all the fields on the page with details from the JSON answer of the API.
+ * @param {JSON} bookJSON 
+ */
+
+function displayDetails(bookJSON) {
+  const volumeInfo = bookJSON.volumeInfo;
+  document.getElementById('title-header').innerText = volumeInfo.title;
+  document.getElementById('authors').innerText = 
+    `Authors: ${volumeInfo.authors}`;
+  document.getElementById('published-date').innerText = 
+    `Published date: ${volumeInfo.publishedDate}`;
+  document.getElementById('category').innerText =
+    `Categories: ${volumeInfo.categories}`;
+  document.getElementById('publisher').innerText =
+    `Publishing house: ${volumeInfo.publisher}`;
+  document.getElementById('page-count').innerText =
+    `Page count: ${volumeInfo.pageCount}`;
+  document.getElementById('description').innerText = volumeInfo.description;
+  document.getElementById('identifiers').innerText = getStringIdentifiers(volumeInfo.industryIdentifiers);
+}
+
+/**
+ * identifiers array ontains objects like:
```

Comment:
I know it's just a comment but  you can change it to "`identifiers array contains objects like:"`

Reply:
Oops my bad.